HousingRepairsSchedulingApi/Services/Drs/DrsService.cs

```diff
@@ -10,7 +10,7 @@ namespace HousingRepairsSchedulingApi.Services.Drs
 
     public class DrsService : IDrsService
     {
-        private const string DrsContract = "0";
+        private const string DrsContract = "H01";
```

Comment:
what does H01 represent?

Reply:
"H01" represents a general building repair and is the contract value the PO (Lindsay Matthews) said should be used.